algorithms/gr-dij/csharp/DijkstraAlgorithm.cs

```diff
@@ -0,0 +1,92 @@
+// A C# program for Dijkstra's  shortest path algorithm. 
+using System; 
+
+class Dijkstra { 
+
+	static int V = 9; 
+
+	static int minDistance(int[] dist, bool[] sptSet) 
+	{ 
+		// Initialize min value 
+		int min = int.MaxValue, min_index = -1; 
+
+		for (int v = 0; v < V; v++) 
+		{
+			if (sptSet[v] == false && dist[v] <= min) 
+			{ 
+				min = dist[v]; 
+				min_index = v; 
+			} 
+		}
+
+		return min_index; 
+	} 
+
+	// Function to print the constructed distance array 
+	static void printSolution(int[] dist) 
+	{ 
+		Console.Write("Vertex \t\t Distance "
+					+ "from Source\n"); 
+		for (int i = 0; i < V; i++) 
+		{
+			Console.Write(i + " \t\t " + dist[i] + "\n"); 
+		}
+	} 
+
+	// Funtion that implements Dijkstra's shortest path algorithm 
+	static void Dodijkstra(int[, ] graph, int src) 
+	{ 
+		int[] dist = new int[V];  
+
+		
+		bool[] sptSet = new bool[V]; 
+
+	
+		for (int i = 0; i < V; i++) 
+		{ 
+			dist[i] = int.MaxValue; 
+			sptSet[i] = false; 
+		} 
+
+		
+		dist[src] = 0; 
+
+		// Find shortest path for all vertices 
+		for (int count = 0; count < V - 1; count++) 
+		{ 
+			
+			int u = minDistance(dist, sptSet); 
+
+			// Mark the picked vertex as processed 
+			sptSet[u] = true; 
+
+			// Update dist value of the adjacent 
+			// vertices of the picked vertex. 
+			for (int v = 0; v < V; v++) 
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6349348)